mmv1/third_party/terraform/tests/resource_gke_hub_feature_membership_test.go.erb

```diff
@@ -573,6 +573,204 @@ resource "google_gke_hub_feature_membership" "feature_member" {
+    }
+  }
+  provider = google-beta
+}
+`, context)
+}
+
+func testAccGkeHubFeatureMembership_gkehubFeatureAcmOciUpdate(context map[string]interface{}) string {
+  return gkeHubFeatureProjectSetup(context) + gkeHubClusterMembershipSetup(context) + Nprintf(`
+resource "google_gke_hub_feature" "feature" {
+  project = google_project.project.project_id
+  name = "configmanagement"
+  location = "global"
+
+  labels = {
+    foo = "bar"
+  }
+  provider = google-beta
+  depends_on = [google_project_service.container, google_project_service.gkehub]
+}
+
+resource "google_service_account" "feature_sa" {
+  project = google_project.project.project_id
+  account_id = "feature-sa"
+  provider = google-beta
+}
+
+resource "google_gke_hub_feature_membership" "feature_member_1" {
+  project = google_project.project.project_id
+  location = "global"
+  feature = google_gke_hub_feature.feature.name
+  membership = google_gke_hub_membership.membership.membership_id
+  configmanagement {
+    version = "1.12.0"
+    config_sync {
+      source_format = "hierarchy"
+      oci {
+        sync_repo = "us-central1-docker.pkg.dev/sample-project/config-repo/config-sync-gke:latest"
+        policy_dir = "config-sync"
+        sync_wait_secs = "15"
+        secret_type = "gcenode"
+        gcp_service_account_email = google_service_account.feature_sa.email
+      }
+      prevent_drift = true
+    }
+    policy_controller {
+      enabled = true
+      audit_interval_seconds = "100"
+      exemptable_namespaces = ["onetwothree", "fourfive"]
+      template_library_installed = true
+      referential_rules_enabled = true
+      log_denies_enabled = true
+    }
+  }
+  provider = google-beta
+}
+`, context)
+}
+
+func testAccGkeHubFeatureMembership_gkehubFeatureAcmOciRemoveFields(context map[string]interface{}) string {
+  return gkeHubFeatureProjectSetup(context) + gkeHubClusterMembershipSetup(context) + Nprintf(`
+resource "google_gke_hub_feature" "feature" {
+  project = google_project.project.project_id
+  name = "configmanagement"
+  location = "global"
+
+  labels = {
+    foo = "bar"
+  }
+  provider = google-beta
+  depends_on = [google_project_service.container, google_project_service.gkehub]
+}
+
+resource "google_service_account" "feature_sa" {
+  project = google_project.project.project_id
+  account_id = "feature-sa"
+  provider = google-beta
+}
+
+resource "google_gke_hub_feature_membership" "feature_member_1" {
+  project = google_project.project.project_id
+  location = "global"
+  feature = google_gke_hub_feature.feature.name
+  membership = google_gke_hub_membership.membership.membership_id
+  configmanagement {
+    version = "1.12.0"
+    policy_controller {
+      enabled = true
+      audit_interval_seconds = "100"
+      exemptable_namespaces = ["onetwothree", "fourfive"]
+      template_library_installed = true
+      referential_rules_enabled = true
+      log_denies_enabled = true
+    }
+  }
+  provider = google-beta
+}
+`, context)
+}
```

Comment:
This is a challenging PR to review due to the tests being unable to run in CI. Were you able to run these tests locally and get them to pass. I've been trying to run them locally but running into failures on my end.

Reply:
Tested the changes in local and pasting the Acceptance Test results.

```
==> Checking that code complies with gofmt requirements...
go vet
go generate  ./...
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google-beta -v -run=TestAccGkeHubFeatureMembership_gkehubFeatureAcmOci -timeout 240m -ldflags="-X=github.com/hashicorp/terraform-provider-google-beta/version.ProviderVersion=acc"
=== RUN   TestAccGkeHubFeatureMembership_gkehubFeatureAcmOci
=== PAUSE TestAccGkeHubFeatureMembership_gkehubFeatureAcmOci
=== CONT  TestAccGkeHubFeatureMembership_gkehubFeatureAcmOci
--- PASS: TestAccGkeHubFeatureMembership_gkehubFeatureAcmOci (956.07s)
PASS
ok  	github.com/hashicorp/terraform-provider-google-beta/google-beta	956.168s

```